src/style.css

```
@@ -0,0 +1,90 @@
+/* stylelint-disable color-hex-length */
+/* stylelint-disable color-hex-case */
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+.bigOne {
+  display: block;
+  margin-top: 20px;
+}
+
+.smallOne {
+  display: block;
+  margin-top: 20px;
+}
+
+.bigBar {
+  padding-left: 62px;
+  font-family: Avenir, Helvetica, sans-serif;
```

Better to inherit this font
```suggestion
  font-family: inherit;
```